packages/ast-spec/src/expression/TaggedTemplateExpression/spec.ts

```diff
@@ -1,12 +1,13 @@
 import type { AST_NODE_TYPES } from '../../ast-node-types';
 import type { BaseNode } from '../../base/BaseNode';
+import type { ChainExpression } from '../../expression/ChainExpression/spec';
 import type { TSTypeParameterInstantiation } from '../../special/TSTypeParameterInstantiation/spec';
 import type { Expression } from '../../unions/Expression';
 import type { TemplateLiteral } from '../TemplateLiteral/spec';
 
 export interface TaggedTemplateExpression extends BaseNode {
   type: AST_NODE_TYPES.TaggedTemplateExpression;
   quasi: TemplateLiteral;
-  tag: Expression;
+  tag: Exclude<Expression, ChainExpression>;
```

Comment:
generally we prefer to avoid using `Exclude` in our AST as it makes it complicated to understand.

in this case `Expression` is "good enough" to use -- it already intentionally includes a few nodes that aren't strictly expressions
https://github.com/typescript-eslint/typescript-eslint/blob/16c344ec7d274ea542157e0f19682dd1930ab838/packages/ast-spec/src/unions/Expression.ts#L38-L42

So let's just leave it as `Expression`